ng5/src/app/screens/auth/screens/book-detail/book-detail.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { BookService } from '../../../../models/book.service';
+
+@Component({
+  selector: 'app-book-detail',
+  templateUrl: './book-detail.component.html',
+  styleUrls: ['./book-detail.component.scss']
+})
+export class BookDetailComponent implements OnInit {
+
+  id: string;
```

Comment:
instead of doing this make a class Book and add a single property to this component: book: Book